CONTRIBUTING.md

```diff
@@ -70,11 +70,11 @@ The project is organized as follows:
 
 - `core/` - Core functionality and CLI application
   - `src/` - Core source code
```

Comment:
More of these paths need to be updated as well. See the proposal at https://github.com/Azure/azure-mcp-pr/issues/282#issuecomment-3198615547